docs/en/concepts-and-designs/kubesphere-monitoring.md

```diff
@@ -0,0 +1,43 @@
+# KubeSphere Monitoring
+
+## Overview
+
+The KubeSphere monitoring system comprises many components that work together to achieve overall monitoring functionality. KubeSphere defaults to installing the monitoring module. It will deploy  or create a few components within the namespace `kubesphere-monitoring-system`:
+
+- Prometheus Operator: automates tasks for operating Prometheus instances.
+- Prometheus Servers: scrapes metrics and provides monitoring service.
+- Kube-state-metrics: exposes kubernetes resource metrics.
+- Node-exporter: exposes node metrics.
+
+Besides, the monitoring stack includes CRD resources:
+
+- ServiceMonitor: defines monitoring for a set of services.
+- Prometheus: defines Prometheus deployments, `k8s` and `k8s-system`.
+- PrometheusRule: describes recording rules.
+
+There are two independent Prometheus servers implemented as StatefulSet, `prometheus-k8s` and `prometheus-k8s-system`, monitoring different services.
+
+The first monitors:
+
+- kube-state-metrics
+- node-exporter
+- kubelet
+- s2i-operator
+- prometheus-k8s-system
+
+The latter monitors:
+
+- etcd
+- coredns
+- kube-apiserver
+- kube-scheduler
+- kube-controller-manager
+- prometheus-k8s
+
+## Setup
+
```

Comment:
Here we can add a table , the columes include links for kube-prometheus , kubernetes-mixin, the row include ks versions

Reply:
i think we can link to the latest dependencies because we are on the master branch. The guide can be updated as the new ks is released